soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3724,7 +3737,7 @@ void DrawRandoEditor(bool& open) {
     const char* randoGanonsTrial[3] = { "Skip", "Set Number", "Random Number" };
 
     // World Settings
-    const char* randoStartingAge[3] = { "Child", "Adult", "Random" };
+    const char* randoStartingAge[3] = { "Adult", "Child", "Random" };
     const char* randoShuffleEntrances[2] = { "Off", "On" };
```

Comment:
We should keep this the same order as this was before. Most options we try to go from the most vanilla option towards whatever is further away from vanilla (bar some exceptions). This will require also changing the order on the 3DS side (both in the Option class and the enum that's used for this option).